utils/kata-manager.sh

```diff
@@ -720,26 +721,74 @@ handle_containerd()
 	containerd --version
 }
 
+handle_docker()
+{
+	{ containerd_installed; ret=$?; } || true
+	if [ "$ret" -eq 0 ]
+	then
+		info "Backing up previous containerd configuratinn"
+		local cfg="/etc/containerd/config.toml"
```

Comment:
Actually, could you make this a global read-only as it's also referenced in `configure_containerd()`.